src/Concordium/Client/Runner.hs

```diff
@@ -1257,6 +1257,15 @@ bakerAddTransaction baseCfg txOpts f batcBakingStake batcRestakeEarnings confirm
           Just x -> return x
           Nothing -> logFatal [err]
 
+-- |Query the chain for the minimum baker stake threshold. Fail if the chain cannot be reached.
+getBakerStakeThresholdOrDie :: ClientMonad IO Types.Amount
+getBakerStakeThresholdOrDie = do
+  blockSummary <- getFromJson =<< withBestBlockHash Nothing getBlockSummary
+  case blockSummary of
+    Nothing -> do
+      logFatal ["No Block Found"]
```

Comment:
Please make a better error message. Perhaps
```
Could not reach the node to retrieve baker stake threshold.
```

Reply:
Sounds good. I had matched the message used elsewhere, but I've now clarified those as well.